docker/docker-compose.yml

```diff
@@ -42,7 +39,7 @@ services:
         COINGECKO_ENDPOINT: https://pro-api.coingecko.com/api/v3/coins/markets?vs_currency=usd&ids=bitcoin
         COINGECKO_KEY: "TBD" #PLEASE FILL IT WITH YOUR OWN
         MONITOR_WITHDRAWALS_CRON: "0 */1 * * * ?" # Every Minute
-        ENABLE_REMOTE_FM_SIGNER: "false" #Changes the way of signing, instead of a auto-generated internal wallet, the FM will sign with a remote signer function on AWS Lambda
+        ENABLE_REMOTE_SIGNER: "false" #Changes the way of signing, instead of a auto-generated internal wallet, the FM will sign with a remote signer function on AWS Lambda
```

Comment:
FM still in the comment

Reply:
```suggestion
        ENABLE_REMOTE_SIGNER: "false" #Changes the way of signing, instead of a auto-generated internal wallet, NodeGuard will sign with a remote signer function on AWS Lambda
```